phpBB/language/en/install.php

```diff
@@ -530,6 +532,27 @@
 	'UPDATE_DATABASE_SCHEMA'		=> 'Updating database schema',
 	'UPDATE_FILES'					=> 'Update files',
 	'UPDATE_FILES_NOTICE'			=> 'Please make sure you have updated your board files too, this file is only updating your database.',
+	'UPDATE_INCOMPLETE'				=> 'Your phpBB installation has not been correctly updated.',
```

Comment:
> All of this here seems like a verbose mess of information not really needed.

Why don't write an explicative text? This can be benefit for administrators that don't have much knowledge of phpBB. Or there is some reason to not include this text?

> So just provide a link to the update script at root/install/database_update.php

Yes, but the `install` folder may have been deleted and some administrators can be novice, so I think that we need to explain to the user, the complete way to retrieve the archive and finish the update.